plugins/status/plugin.go

```diff
@@ -195,14 +198,20 @@ func (p *Plugin) loop() {
 }
 
 func (p *Plugin) oneShot(ctx context.Context) error {
-
 	req := &UpdateRequestV1{
 		Labels:    p.manager.Labels(),
 		Discovery: p.lastDiscoStatus,
 		Bundle:    p.lastBundleStatus,
 		Bundles:   p.lastBundleStatuses,
 	}
 
+	globalMetrics, err := metrics.GlobalMetricsRegistry.Gather()
```

Comment:
The `oneShot` only happens after a bundle status update happens or reconfiguration of the status plugin. That might mean on systems that don't have much churn on bundles we wouldn't get these metrics from the status API very often. Does it make sense to maybe add a timer or something to trigger this periodically even if no bundle status changes have happened?

Reply:
the idea was to include metrics with each status updates, not to make more frequent updates. AFAIK the bundles are updated each 30 seconds or so and each such update causes a status update. I.e. we already have such timer in some way. At least I'm seeing periodic status updates on the server

Reply:
Ahh ok, yea, that makes sense. I was thinking the status updates only went out if the content changed but we still call the listeners on the bundle plugin even if its the same etag.